src/main/java/ca/concordia/soen/ThrowsGenericVisitor.java

```diff
@@ -13,71 +14,50 @@ public ThrowsGenericVisitor(CompilationUnit compilationUnit) {
     public boolean visit(MethodDeclaration node) {
         List<Type> methodDeclarationThrownExceptions = node.thrownExceptionTypes();
         Block body = node.getBody();
-        if(methodDeclarationThrownExceptions.size() == 1){
+        if (methodDeclarationThrownExceptions.size() == 1) {
             if (methodDeclarationThrownExceptions.get(0) instanceof SimpleType && methodDeclarationThrownExceptions.get(0).toString().equals("Exception")) {
                 antiPatternDetect(node, methodDeclarationThrownExceptions.get(0).getStartPosition());
             }
-        }
-        else if (methodDeclarationThrownExceptions.size() == 0){
-            if(hasTryCatch(body)){
-                for(Object statement: body.statements()){
+        } else if (methodDeclarationThrownExceptions.size() == 0) {
+            if (hasTryCatch(body)) {
+                for (Object statement : body.statements()) {
```

Comment:
You don't want this for. This is what the TA explained that does not work. So, instead, you want to add a visitor here.  Please check this commit for reference.

Reply:
Changed the code.